Stratis.SmartContracts/ISerializer.cs

```diff
@@ -91,6 +91,16 @@ public interface ISerializer
         /// </summary>
         ulong ToUInt64(byte[] val);
 
+        /// <summary>
+        /// Deserializes the first 16 bytes of a byte array into an UInt128. If the given bytes are null, empty, or deserialization fails, returns default(UInt128).
+        /// </summary>
+        UInt128 ToUInt128(byte[] val);
```

Comment:
Still `Serialize` methods missing for `UInt128` and `UInt256`

Reply:
Look here: https://github.com/stratisproject/StratisFullNode/pull/365/files?file-filters%5B%5D=#diff-de42f079363455aa4a9a840704eb10090823a7fb1a3984dce43511a5d87dbba6

Reply:
I look at there but i didn't understand still :). Those methods aren't necessary in here ?

Reply:
Ok I see what you mean now. We have to add that in the next PR.